service/controller/resource/masters/create.go

```diff
@@ -23,7 +23,6 @@ func (r *Resource) createStateMachine() state.Machine {
 			ProvisioningSuccessful:         r.provisioningSuccessfulTransition,
 			ClusterUpgradeRequirementCheck: r.clusterUpgradeRequirementCheckTransition,
 			MasterInstancesUpgrading:       r.masterInstancesUpgradingTransition,
-			WaitForMastersToBecomeReady:    r.waitForMastersToBecomeReadyTransition,
```

Comment:
Master instances upgrade process doesn't proceed if nodes aren't `Ready` and VMs in `Successful` state so this is not needed anymore.